src/Commands/MeltorCommand.php

```diff
@@ -0,0 +1,309 @@
+                if ($this->separateForeignKeys) {
+                    $columnConstraintMigrations[] = $foreignKeyMigration;
+                } else {
+                    $columnMigrations[] = $foreignKeyMigration;
+                }
+            }
+
+            $tableMigrations[$tableName]['laravel'] = $columnMigrations;
+            $tableMigrations[$tableName]['raw']     = $columnMigrationsRaw;
+
+            if ($this->separateForeignKeys) {
+                $constraintMigrations[$tableName] = $columnConstraintMigrations;
+            }
+        }
+
+        $migrationCode      = $this->meltor->generateMigration($tableMigrations, $this->meltor->getMigrationTemplate('migrationComment'), $constraintMigrations, $rawExceptions);
+        $migrationCode      = $this->meltor->beautify($migrationCode);
+        $migrationFilePath  = $this->meltor->writeMigration($migrationCode, $this->meltor->config('migration.name'), $this);
+        $showDisclaimerText = true;
+
+        if ($this->option('testrun')) {
+            $showDisclaimerText = $this->testrun();
+        }
+
+        if ($showDisclaimerText) {
+            $this->newLine();
+            $this->line(
+                sprintf('Test and inspect "%s" to make sure the migration fits your requirements', $migrationFilePath)
+            );
+            $this->line('You can then delete all prior migrations and commit');
+            $this->warn('It may be necessary to keep migrations that alter tables created by frameworks or packages!');
+        }
+
+        if ($this->warnAboutFloat) {
+            $this->warn('FLOAT columns may be turned to DOUBLE by Laravel!');
+        }
+
+        $this->newLine();
+
+        return 0;
+    }
+
+    /**
+     * Compares the database structure before and after the new migration.
+     *
+     * Returns true success, but also if there are still changes to review.
+     * Returns false on complete failures.
+     *
+     * @return bool
+     */
+    protected function testrun(): bool
+    {
+        $success                 = false;
+        $beforeStructureFileName = $this->meltor->combinePath(
+            $this->meltor->config('testrun.folder'),
+            $this->meltor->config('testrun.beforeStructureFileName')
+        );
+        $afterStructureFileName  = $this->meltor->combinePath(
+            $this->meltor->config('testrun.folder'),
+            $this->meltor->config('testrun.afterStructureFileName')
+        );
+
+        // Reading existing database structure.
+        $this->newLine();
+        $this->line('Performing structure comparison test run');
+        $this->meltor->writeStructureDump($beforeStructureFileName, $this);
+
+        // Backup existing database.
+        $this->newLine();
+        $this->line('Backing up database before applying temporary changes');
+        $this->call('protector:export', ['--file' => $this->meltor->config('testrun.backupFileName')]);
```

Comment:
If this fails, it will just continue anyways. Maybe you want to check the result and/or the existence of the backup file?